src/commands/templates.rs

```diff
@@ -81,6 +81,11 @@ pub struct Install {
     /// If present, updates existing templates instead of skipping.
     #[clap(long = "upgrade", alias = "update")]
     pub update: bool,
+
+    /// By default, Spin upgrades all the templates in the source.
+    /// Pass this flag with a list of template IDs to update specific ones.
+    #[clap(long, value_delimiter = ',')]
```

Comment:
I don't love the comma separator - I think we usually use allow-multiple args for this sort of thing?  (But I may be wrong!)

Reply:
I do not have a really strong preference but having to type `--id template-a --id template-b` seemed a bit more verbose but that does seem more semantically correct with it being `id` (singular)